disputer/disputer.js

```diff
@@ -169,9 +165,6 @@ class Disputer {
         disputeResult: logResult
       });
     }
-
-    // Update the EMP Client since we disputed liquidations.
-    await this.empClient.update();
```

Comment:
these update calls I removed as they are redundant in this bot. update is always called at the start of the operation so no need to update it after polling action is finished.